GenerateSDK.md

```diff
@@ -14,22 +14,22 @@ This repo contains commercetools SDKs and this is guide for how to generate the
 
 ## Generating the sdks
 
-run `yarn generate` to generate the sdks. (make sure the environement variables are set first)
+run `yarn generate` to generate the sdks. (make sure the environment variables are set first)
 
 ## Environment variables
 
-| Variable name |                                                    Description                                                    |
-| ------------- | :---------------------------------------------------------------------------------------------------------------: |
-| ML_API_REF    | The path to commercetools [machine learning RAML api](https://github.com/commercetools/ml-services-api-reference) |
-| API_RAML_FILE |        The path to commercetools [RAML api](https://github.com/commercetools/commercetools-api-reference)         |
+| Variable name |                                                         Description                                                          |
+| ------------- | :--------------------------------------------------------------------------------------------------------------------------: |
+| ML_API_REF    | The path to Composable Commerce [Machine Learning API reference](https://github.com/commercetools/ml-services-api-reference) |
+| API_RAML_FILE |           The path to Composable Commerce [RAML API](https://github.com/commercetools/commercetools-api-reference)           |
```

Comment:
Sorry I must have missed this in the first round 🙈  I think it's better to say API reference because RAML might go away at some point
```suggestion
| API_RAML_FILE |           The path to Composable Commerce [HTTP API reference](https://github.com/commercetools/commercetools-api-reference)           |
```